src/style-spec/reference/v8.json

```diff
@@ -2524,6 +2500,72 @@
     "value": "*",
     "doc": "A filter selects specific features from a layer."
   },
+  "filter_symbol": {
+    "type": "boolean",
+    "doc": "Expression which determines whether or not to display a symbol. Symbols support `dynamic-filtering`, meaning this expression can use the `[\"pitch\"]` and `[\"distance-from-center\"]` expressions to reference the current state of the view.",
```

Comment:
Nit:
Is `dynamic-filtering` a string that users will need to type? If not, I think should just be written in plain text as dynamic filtering.

I'm noticing that we have an inconsistency in the style-spec for whether to write expressions as: `expression-name`,  `'expression-name'`, [\"expression-name\"], or `[\"expression-name\"]`. I'd love to hear from the docs team on this.